app/default.settings.js

```diff
@@ -117,6 +117,14 @@ drupalgap.settings.locale = {
 drupalgap.settings.title = 'DrupalGap';
  
 // App Front Page
+
```

Comment:
@mattshoaf This is interesting. I believe this would be better placed in the docs as an example. As such an edge case, I worry it would clutter the `default.settings.js` file. It would probably go best as a new page (or added to an existing page) within this folder: https://github.com/signalpoint/DrupalGap/tree/7.x-1.x/docs/09_Pages/Navigating_Pages